fetch.bs

```diff
@@ -4482,6 +4492,11 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
  <li><p>If <i>authentication-fetch flag</i> is set, then create an <a>authentication entry</a>
  for <var>request</var> and the given realm.
 
+ <li><p>If <i>revalidatingFlag</i> is set and <var>httpRequest</var>'s <a for=request>cache mode</a>
+ is "<code>async-stale-revalidated</code>" then asynchronously perform a
+ <a>HTTP-network-or-cache fetch</a> using <var>request</var> with <a for=request>cache mode</a>
```

Comment:
This will bypass the service worker. Is that okay? It seems problematic that you can create a network request like that.

Reply:
SwR is a concept of HTTP cache, which basically sits below the service worker, so service workers being not able to observe revalidation requests sounds somewhat natural to me from implementation pov.  That's said... in this way service workers may keep holding stale responses in cache storage, and now that we're talking about Fetch layer's SwR handling, it might be worth discussing.